diff -urp aide-0.14.orig/src/be.c aide-0.14/src/be.c
--- aide-0.14.orig/src/be.c	2010-05-18 12:36:19.000000000 -0400
+++ aide-0.14/src/be.c	2010-05-18 12:39:53.000000000 -0400
@@ -161,7 +161,7 @@ FILE* be_init(int inout,url_t* u,int isz
 #endif
 #ifdef WITH_ZLIB
     if(iszipped && !inout){
-      fh=gzdopen(fd,"wb9+");
+      fh=gzdopen(fd,"wb9");
       if(fh==NULL){
 	error(0,_("Couldn't open file %s for %s"),u->value,
 	      inout?"reading\n":"writing\n");
diff -urp aide-0.14.orig/src/commandconf.c aide-0.14/src/commandconf.c
--- aide-0.14.orig/src/commandconf.c	2010-05-18 12:36:19.000000000 -0400
+++ aide-0.14/src/commandconf.c	2010-05-18 12:36:49.000000000 -0400
@@ -84,7 +84,7 @@ int commandconf(const char mode,const ch
     else {
       tmp=(char*)malloc(sizeof(char)
 			*(strlen(after)+strlen(line)+2));
-      strcat(tmp,after);
+      strcpy(tmp,after);
       strcat(tmp,"\n");
       strcat(tmp,line);
       free(after);
